aspnet/mvc/views/working-with-forms.rst

```diff
@@ -0,0 +1,486 @@
+    <span class="field-validation-valid" data-valmsg-replace="true" 
+     data-valmsg-for="Email"></span><br>
+    Password: <input name="Password" id="Password" type="password" 
+     data-val-required="The Password field is required." data-val="true"><br>
+    <span class="field-validation-valid" data-valmsg-replace="true" 
+     data-valmsg-for="Password"></span><br>
+    <button type="submit">Register</button>
+    <input name="__RequestVerificationToken" type="hidden" value="<removed for brevity>" />
+  </form>
+
+The Select Tag Helper
+-------------------------
+
+.. question One odd thing I noticed about this is that "asp-for" doesn't require the Model. prefix, but "asp-items" does. If anybody knows why this is, I'd love to hear it.  http://www.exceptionnotfound.net/tag-helpers-in-asp-net-5-an-overview/#selecttaghelper 
+
+The `Select Tag Helper <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/TagHelpers/SelectTagHelper/index.html>`__ generates `select <https://www.w3.org/wiki/HTML/Elements/select>`__ and associated `option <https://www.w3.org/wiki/HTML/Elements/option>`__ elements for properties of your model. ``asp-for`` specifies the model property  name for the `select <https://www.w3.org/wiki/HTML/Elements/select>`__ element  and ``asp-items`` specifies the `option <https://www.w3.org/wiki/HTML/Elements/option>`__ elements.  For example:
+
+.. literalinclude::   forms/sample/final/Views/Home/Index.cshtml
+  :language: HTML
+  :lines: 4
+  :dedent: 3
+  
+Consider the following view model:
+
+.. literalinclude::  forms/sample/final/ViewModels/CountryViewModel.cs
+  :language: c#
+  
+The ``Index`` method initializes the ``CountryViewModel``, sets the selected country and passes it to the ``Index`` view.
+
+.. literalinclude:: forms/sample/final/Controllers/HomeController.cs
+  :language: c#
+  :lines: 8-13
+  :dedent: 6
+
+The HTTP POST ``Index`` method displays the selection:
+
+.. literalinclude:: forms/sample/final/Controllers/HomeController.cs
+  :language: c#
+  :lines: 15-27
+  :dedent: 6
+  
+The ``Index`` view:
+
+.. literalinclude::   forms/sample/final/Views/Home/Index.cshtml
+  :language: HTML
+  :emphasize-lines: 4
+  
+Which generates the following HTML:
+
+.. code-block:: HTML  
+  :linenos:
+  :emphasize-lines: 2-6 
+
+  <form method="post" action="/">
+    <select id="Country" name="Country">
+      <option value="MX">Mexico</option>
+      <option selected="selected" value="CA">Canada</option>
+      <option value="US">USA</option>
+    </select> 
+      <br /><button type="submit">Register</button>
+    <input name="__RequestVerificationToken" type="hidden" value="<removed for brevity>" />
+  </form>
+
+:Note: We do not recommend using ``ViewBag`` or ``ViewData`` with the Select Tag Helper. A view model is more robust at providing MVC metadata and generally less problematic. 
+
+The ``asp-for`` attribute value doesn't require a ``Model`` prefix, while ``asp-items does``:
```

Comment:
- See earlier comments on how different `asp-for` is from everything else.
- Remove "does" from the `asp-items` code block.